lib/cuisines.rb

```diff
@@ -0,0 +1,5 @@
+module Cuisines
+  CUISINES = { Italian: { spaghetti: 5, pizza: 3, gelato: 2},
+               American: { hotdog: 1, burger: 5, coke: 0.5 },
+               Chinese: { fried_rice: 2, peking_duck: 7, sweet_and_sour_chicken: 3} }
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)